scripts/tests/harness.sh

```diff
@@ -80,6 +80,21 @@ function forest_print_logs_and_metrics {
   cat "$LOG_DIRECTORY"/*
 }
 
+function wait_until_rpc_is_ready {
```

Comment:
Same here. A sane timeout is always helpful.

Reply:
This has been removed